Collectors/UdpCollector.py

```diff
@@ -272,6 +272,12 @@ def on_message(channel, method, properties, body):
             # Send to message queue
             # Get the address and port
             addr = loaded_json['remote'].rsplit(":", 1)
+
+            # High water mark for the queue
+            if len(message_q.qsize() > config.get('AMQP', 'high_water_mark', fallback=100)):
+                while message_q.qsize() > config.get('AMQP', 'low_water_mark', fallback=10):
```

Comment:
Nitpicky change: make the config knobs separate from the `AMQP` block, since it affects internal processing, and not the MQ size. (A better name than `Worker` would be appreciated.)
```suggestion
            if len(message_q.qsize() > config.get('Worker', 'high_water_mark', fallback=100)):
                while message_q.qsize() > config.get('Worker', 'low_water_mark', fallback=10):
```